ui/src/app/utils/stores.tsx

```diff
@@ -0,0 +1,23 @@
+import {useEffect, useState} from 'react';
+import {atom, Atom} from './subscribable';
+
+interface RouteDataStore {
```

Comment:
This gets killed after migrating off Angular right? Maybe leave a comment here so we can remember what to rip out afterwards.